share/smack/include/math/math.h

```diff
@@ -58,30 +55,24 @@ double modf(double x, double* y);
 double copysign(double x, double y);
 double nan(const char* x);
 int __isnormal(double x);
-int __isSubnormal(double x);
+int __issubnormal(double x);
 int __iszero(double x);
 int __isinf(double x);
 int __isnan(double x);
 int __isnegative(double x);
-int __ispositive(double x);
 int __signbit(double x);
-int signbit(double x);
 int __fpclassify(double x);
-int fpclassify(double x);
-int __finite(double x);
+int __isfinite(double x);
 
 //long doubles
```

Comment:
What do we do with long doubles?

Reply:
SMACK currently doesn't support the long double functions.